source/includes/harvest/_tags.md

```diff
@@ -0,0 +1,165 @@
+# Tags
+
+Tags applied to a Greenhouse model, like candidate tags
+
+## The candidate tag object
+
+```json
+{
+  "id": 230,
+  "name": "New York",
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The candidate tag's unique identifier |
+| name | The name of the tag |
+
+## Candidate Tags
+
+List all of an organization's candidate tags
+
+```shell
+curl -X GET 'https://harvest.greenhouse.io/v1/tags/candidate'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 365,
+    "name": "Ruby"
+  },
+  {
+    "id": 366,
+    "name": "Rails"
+  },
+  {
+    "id": 367,
+    "name": "Junior"
+  }
+]
+```
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/tags/candidate`
+
+### Querystring parameters
+
+No query string parameters.
```

Comment:
If there are no querystring params for some of these endpoints, maybe we could just remove these empty sections?